packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/amplify-meta-updaters.ts

```diff
@@ -106,20 +106,30 @@ export const getPostUpdateAuthMetaUpdater = (context: any) => async (resourceNam
   return resourceName;
 };
 
-function getFrontendConfig(authParameters: AuthParameters) {
-  const loginMechanisms = (authParameters?.aliasAttributes || []).map((att: string) => att.toUpperCase());
+export function getFrontendConfig(authParameters: AuthParameters) {
   const verificationMechanisms = (authParameters?.autoVerifiedAttributes || []).map((att: string) => att.toUpperCase());
+  const loginMechanisms: Set<string> = new Set<string>();
```

Comment:
Not a big deal, but you don't need `Set<string>` on both sides of the assignment. I would make this:

```js
const loginMechanisms = new Set<string>();
```